src/main/resources/db/migration/common/V1_233__v53_updates.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE transcription ADD COLUMN hearing_date_tmp DATE;
+UPDATE transcription SET hearing_date_tmp = CAST(hearing_date AS DATE);
+ALTER TABLE transcription DROP COLUMN hearing_date;
+ALTER TABLE transcription RENAME COLUMN hearing_date_tmp TO hearing_date;
+ALTER TABLE transcription ADD COLUMN is_manual boolean NOT NULL DEFAULT FALSE;
```

Comment:
Ideally the old staging data would have the transcriptions updated to be is_manual=TRUE where the transcription_type is `999	Other`

Reply:
Sorry - yep it was supposed to be NOT 999